docs/conf.py

```diff
@@ -32,7 +32,7 @@
 project = f'Envoy Gateway {version}'
 author = 'Envoy Gateway Project Authors'
 
-copyright = '2022 Envoy Gateway Project Authors | ' + fullversion
+copyright = '2022 Envoy Gateway Project Authors | <a href="https://github.com/envoyproxy/gateway">GitHub</a> | ' + fullversion
```

Comment:
Please remove `2022` so we don't have to manage the copyright on a yearly basis.